src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -7476,6 +7480,23 @@ export class StudyViewPageStore {
         default: {},
     });
 
+    private calculateSampleCount(
```

Comment:
this is a pure function (no reference to this), so might as well just have this in module scope (not a method).